app/models/order.rb

```diff
@@ -14,9 +14,17 @@ def grand_total
   def total_quantity
     item_orders.sum('quantity')
   end
-  
+
   def status
     return "Packaged" if self.item_orders.where('status = ?', 'Pending').count == 0
     "Pending"
   end
+
+  def item_quantity(item)
+    ItemOrder.where("order_id = ?", self.id).where("item_id = ?", item.id).first.quantity
```

Comment:
this totally makes sense, but I didn't realize you could call where multiple times. OMG yay